### PR TITLE
Update wancloud.yml

### PR DESCRIPTION
**Pull request by ahlifei**

### CODE REVIEW

File: block-producers/wancloud.yml [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
+block_signing_key: EOS7wUEGtMFCVo4QA3Rica9wEsuVz6DAtEyNFyah22SmuLqxVcdrt
 
 # Primary Contact (Required)
 telegram_user: ahlifei
 keybase_user: ahlifei
 
 # Server Config (Optional)
-domain: 128.1.133.206
-http: 8888
+domain: eosnet.wancloud.cloud
```
Comment (DenisCarriere) [inline block-producers/wancloud.yml]: Server is not pointing to Scholar Testnet, please use the following config:

https://github.com/ScholarTestnet/scholar-server-config

Comment (ahlifei) [reply block-producers/wancloud.yml]: I  have  update,  {
server_version: "d9ad8eec",
head_block_num: 254527,
last_irreversible_block_num: 254400,
head_block_id: "0003e23f0e12c60c82b44d126e7fc899ff14876050b08f713b9a89e6066a2807",
head_block_time: "2018-04-18T07:15:55",
head_block_producer: "staked"
}

Comment (DenisCarriere) [reply block-producers/wancloud.yml]: Still the wrong server config, this points to EOSNet

Comment (ahlifei) [reply block-producers/wancloud.yml]: maybe  yiour  expoloer  has  cache.  may  you  have  a  try  to  clear  the  cache.
ping eosnet.wancloud.cloud
PING eosnet.wancloud.cloud (128.1.134.25) 56(84) bytes of data.
64 bytes from 128.1.134.25 (128.1.134.25): icmp_seq=1 ttl=45 time=44.8 ms
64 bytes from 128.1.134.25 (128.1.134.25): icmp_seq=2 ttl=45 time=35.8 ms
this  return is  ok.

**Pull request closed without merging**